web/vital_records/forms/common.py

```diff
@@ -91,6 +91,8 @@
     (12, "12 - December"),
 ]
 
+NUMBER_CHOICES = [(1, "1"), (2, "2"), (3, "3"), (4, "4"), (5, "5"), (6, "6"), (7, "7"), (8, "8"), (9, "9"), (10, "10")]
```

Comment:
@lalver1 @thekaveman What do y'all think about just deleting this entirely?

Reply:
I think we can do that @angela-tran. In my mind it was only left here in case we ever would want to bring back the feature of having more than 1 copy, but thinking more about it, we should probably just remove it since it is not being used.